ECK Kibana secure settings and encryption refinement (#6429)
elastic/docs-content: deploy-manage/security/k8s-secure-settings.md

     --from-literal=xpack.security.encryptionKey=94d2263b1ead716ae228277049f19975aff864fb4fcfe419c95123c1e90938cd
    ```

    :::{warning}

>     Always ensure the setting is listed in the [configuration reference](kibana://reference/configuration-reference.md) and that the value you provide matches the expected format before adding it to the keystore. Invalid, unsupported, or incorrect settings cause {{kib}} to fail to start.

Reviewer: I think there's a typo here in the last sentence `to fail to start`. Or it just reads odd.

Reviewer: Maybe something like:

```suggestion
    Always ensure the setting is listed in the [configuration reference](kibana://reference/configuration-reference.md) and that the value you provide matches the expected format before adding it to the keystore. Specifying invalid, unsupported, or incorrect settings will prevent {{kib}} from starting successfully.
```

>     Always ensure the setting is listed in the [configuration reference](kibana://reference/configuration-reference.md) and that the value you provide matches the expected format before adding it to the keystore. Specifying invalid, unsupported, or incorrect settings will prevent {{kib}} from starting successfully.

Author: ```suggestion
    Always ensure the setting is listed in the [{{kib}} configuration reference](kibana://reference/configuration-reference.md) and that the value you provide matches the expected format before adding it to the keystore. Specifying invalid, unsupported, or incorrect settings will prevent {{kib}} from starting successfully.
```
